src/io/flutter/logging/FlutterLogTree.java

```diff
@@ -747,6 +718,8 @@ private void updateCounter() {
   }
 
   public void clearEntries() {
+    // Ensure selections are cleared on empty.
+    getTree().setSelectionRows(new int[]{});
```

Comment:
use 
    this.getTree().clearSelection();

instead

Reply:
Good idea.